db/models.py

```diff
@@ -19,8 +19,8 @@ def __str__(self) -> str:
 class Movie(models.Model):
     title = models.CharField(max_length=255)
     description = models.TextField()
-    actors = models.ManyToManyField(Actor)
-    genres = models.ManyToManyField(Genre)
+    actors = models.ManyToManyField(Actor, related_name="actors")
+    genres = models.ManyToManyField(Genre, related_name="genres")
```

Comment:
as it told in the checklist:
better use "movies" related_name

https://github.com/mate-academy/py-movie-moviesession-cinemahall/blob/master/checklist.md